{"stderr":":memory::4:5: error: wrong number of arguments in call. Expected 2 but got 3\n  0003 | func test() {\n> 0004 |     h(f(12), g(4), 12);\n             ^^^^^^^^^^^^^^^^^^\n  0005 | }\n:memory::4:7: error: can not coerce value of type (void) to type bool\n  0003 | func test() {\n> 0004 |     h(f(12), g(4), 12);\n               ^^^^^\n  0005 | }\n:memory::4:5: note: required bool from here\n  0003 | func test() {\n> 0004 |     h(f(12), g(4), 12);\n             ^\n  0005 | }\n:memory::4:14: error: can not coerce value of type (void) to type bool\n  0003 | func test() {\n> 0004 |     h(f(12), g(4), 12);\n                      ^^^^\n  0005 | }\n:memory::4:5: note: required bool from here\n  0003 | func test() {\n> 0004 |     h(f(12), g(4), 12);\n             ^\n  0005 | }\n"}